eng/pipelines/templates/steps/build.yml

```diff
@@ -97,7 +97,7 @@ steps:
           /p:SDKType=${{ parameters.SDKType }} `
           /p:IncludeTests=false `
           /p:PublicSign=false $(VersioningProperties) `
-          /p:Configuration=$(BuildConfiguration) `
+          /p:Configuration=Release `
```

Comment:
I would suggest adding a comment as well so that folks remember why we did this change.

Reply:
Adding one.